config/cli.go

```diff
@@ -51,6 +51,8 @@ type Cli struct {
 	Tags                      map[string]string
 	RetryJoin                 []string
 	EncryptKey                string
+	VodEncryptPublicKey       string
+	VodEncryptPrivateKey      string
```

Comment:
**nit:** IMO either call this `VodDecrypt...` or `VodEncryption...`. Catalyst-api doesn't really encrypt anything on this current version.